docs/changing-behavior-between-calls.md

```diff
@@ -17,17 +19,40 @@ A.CallTo(() => fakeShop.Address).Returns("123 Fake Street").NumberOfTimes(17);
 ```
 
 This could be useful if you want to allow a limited number of calls on
-a [strict fake](strict-fakes.md), but there's a more useful
-application.
+a [strict fake](strict-fakes.md), but there's a more useful application.
+
+## Specifying different behaviors for successive calls
+
+In some cases, you might want to specify different behaviors for successive
+calls to the same method. For instance, in order to test the System Under
+Test's retry logic, a Fake service could be configured to fail once and then
+function properly thereafter. This can be done by chaining behaviors like this:
+
+```csharp
+// Configure the method to throw an exception once, then succeed forever
+A.CallTo(() => fakeService.DoSomething())
+    .Throws<Exception>().Once()
+    .Then
+    .Returns("SUCCESS");
+```
+
+Note that you can only use `Then` after specifying a number of repetitions for

```

Comment:
This is consistent with our current usage of "repeats" and "repetitions", but in light of #808, I wonder if we shouldn't, while we're introducing documentation, look for less controversial terms.
Perhaps something like

> Note that you can only use `Then` after specifying the number of times the behavior should occur.